### DISCUSSION TITLE
is this library for windows and android too?

### DISCUSSION THREAD

**Discussion by MatejMagat305** [Relm4/Relm4#437] [Q&A] [answered]
I m only asking whether windows is suport and whether you planing android app? if yes, maybe would be interasting https://github.com/cnlohr/rawdrawandroid for realy small apk ...

**Comment by AaronErhardt** [comment, marked as answer]
Relm4 is based on GTK which does support Windows but not Android unfortunately: https://relm4.org/book/stable/#platform-support

**Reply by MatejMagat305** [reply]
ou, thank, so if I understand android is not neither planing for far future?

**Reply by AaronErhardt** [reply]
It doesn't look like the GTK devs are planning to add Android support in the near future, which means that Relm4 won't be able to support Android either.

**Reply by jadedevin13** [reply]
@AaronErhardt 
is this possible now?

https://gitlab.gnome.org/GNOME/gtk/-/merge_requests/7555#d819d3917fde8bb39bf3cb421e37c813c5005414

**Reply by AaronErhardt** [reply]
Yes, as long as you have a very recent GTK version available. I don't know about the Rust bindings and the Android development process though, but in general it is possible.
